packages/web3-core-method/src/methods/transaction/GetPendingTransactionMethod.js

```diff
@@ -0,0 +1,53 @@
+/*
+    This file is part of web3.js.
+
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file GetPendingTransactionMethod.js
+ * @author Prince Sinha <sinhaprince013@gmail.com>
+ * @date 2018
+ */
+
+import AbstractMethod from '../../../lib/methods/AbstractMethod';
+
+export default class GetPendingTransactionMethod extends AbstractMethod {
+    /**
+     * @param {Utils} utils
+     * @param {Object} formatters
+     * @param {AbstractWeb3Module} moduleInstance
+     *
+     * @constructor
+     */
+    constructor(utils, formatters, moduleInstance) {
+        super('eth_pendingTransactions', 0, utils, formatters, moduleInstance);
+    }
+
+    /**
+     * This method will be executed after the RPC request.
+     *
+     * @method afterExecution
+     *
+     * @param {Object} response
+     *
+     * @returns {Object}
+     */
+    afterExecution(response) {
+        if (response.length === 0) {
+            return 'No Pending Transactions !!';
```

Comment:
I don't think adding a message for 0 pending transactions is efficient. The return object should just be the response itself.

Reply:
Yes, It could be, I thought that if there will be 0 pending transaction then, it should return a message. Otherwise, it will return an empty array **[ ]**.
@nivida can you comment on this.

Reply:
Returning a empty array is the correct response here. If there is 0 pending transactions then the result is [], it doesn’t matter if it’s empty or not that is the result. Also making other apps have to compare the response against a string is never a good idea.

Reply:
Agreed, Done.